Assets/HoloToolkit/SpatialSound/Scripts/UAudioManager/ActiveEvent.cs

```diff
@@ -61,6 +61,12 @@ public GameObject AudioEmitter
             private set;
         }
 
+        public string MessageToSend
```

Comment:
Right, I think that's in line with my question about ActiveEvent being present in the editor. It seems like right now only the AudioSource would be present in the editor right?